Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/bpmneditor/editor/bpmnio/bpmn-modeler.development.js

```diff
@@ -276,7 +276,7 @@ function forEach(collection, iterator) {
   var convertKey = isArray(collection) ? toNum : identity;
 
   for (var key in collection) {
-    if (has(collection, key)) {
+    if (has(collection, key) && key !== 'remove') {
       val = collection[key];
```

Comment:
Hi Ivo, it could be very tricky to find out where to fix this problem, but is there any other way rather than modifying directly into the source code of bpmn.io? For example, the bpmn.io team do support and answer question, did we search in their forum or their help? In case we have to customize bpmn.io itself, can we modify their source rather than the final bundle file bpmn-modeler.development.js?

Reply:
Yes, this is a temporary solution before we push for the proper one.

Reply:
Please consider whether this is a major and critical issue (showing the editor on Safari) to do this. It would be ok if it's an urgent and serious from a customer perspective.

Reply:
The issue is Safari user can't save BPMN model. The bundle is generated from https://github.com/apromore/QBPSimulator/.

Reply:
The first number I could find online suggested that Safari's market share is ~10% for desktop users.  So that means 10% of our users will be mysteriously unable to work with BPMN.  That seems unacceptable in a release version of the software.